connections.md

```diff
@@ -224,23 +224,58 @@ Possible reasons:
 
 - unknown processing error## 3. Exchange Acknowledgement
 
-TODO: Add an official ACK message.
+## 3. Exchange Complete
 
-After the Response is received, the exchange is technically complete. This remains unconfirmed to the *inviter* however. The *invitee* SHOULD send a message to the *inviter*. As any message will confirm the exchange, any message will do.
+The exchange complete message is used to confirm the exchange to the _inviter_.  This message is required in the flow, as it marks the exchange complete. The _inviter_ may then invoke any protocols desired based on the context expressed via the `pthid` in the DID Exchange protocol.
```

Comment:
```suggestion
The exchange complete message is used to confirm the exchange to the _inviter_. The _inviter_ may then invoke any protocols desired based on the context expressed via the `pthid` in the DID Exchange protocol.
```

Removed because the invitee may instead send the `reuse` message.